README.md

```diff
@@ -17,6 +17,23 @@ Please chat with us on the [discussion board](https://github.com/ethereum-optimi
 
 We welcome your contributions. Read through [CONTRIBUTING.md](./CONTRIBUTING.md) for a general overview of the contributing process for this repository.
 
+### How to Think about Specs
+
+The specs process is a schelling point for where core developers go to modify the OP Stack. A spec serves multiple purposes:
+- Ensures that we know what we are building before we build it
+- Helps code reviewers and auditors understand what the behavior is supposed to be
+- Enables multiple implementations
+
+Some guidelines for writing a good spec:
+- Thoroughly describe the interfaces that other projects may interact with
+- Include a light description of the problem that is being solved and the rationale but the majority of this should be recorded during the design doc phase
```

Comment:
> Include a light description of the problem that is being solved and the rationale but the majority of this should be recorded during the design doc phase

From past conversations, my understanding of design docs is that they are not intended to be deliverables themselves, and they should typically result in some other upstream change. For example, in https://github.com/ethereum-optimism/design-docs/pull/21 we agreed on a new way to version and release L1 contracts, and we [upstreamed](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/VERSIONING.md) it to the monorepo so it's more visible to contributors.

Therefore, I agree the problem and rationale should be recorded during the design doc phase, but I also think that info should still be upstreamed to the specs repo, to act as the permanent, single source of truth. It would provide a worse UX if readers had to jump to design docs—which are likely to become stale as specs are tweaked—to fully understand a spec